packages/paste-website/src/pages/index.tsx

```diff
@@ -1,10 +1,111 @@
 import * as React from 'react';
+import styled from '@emotion/styled';
+import {themeGet} from 'styled-system';
+import {Button} from '@twilio-paste/button';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
 import {SiteWrapper} from '../components/default-site-wrapper';
+import {Grid} from '../components/grid';
+import {StyledHr} from '../components/StyledHr';
+import {Brush} from '../components/icons/Brush';
+import {Terminal} from '../components/icons/Terminal';
+import {WindowLayout} from '../components/icons/WindowLayout';
+
+const LandingContainer = styled(Box)`
+  margin-right: auto;
+  margin-left: auto;
+`;
+
+const StyledButton = styled(Button)`
+  margin-right: ${themeGet('space.space40')};
+  text-decoration: none;
+`;
```

Comment:
This is interesting, I was not expecting this to work _at all_ as we don't expose the classname prop I don't think. If that's true you can extend any component from the system... I'm not sure how I feel about that 🤔

Reply:
I noticed it wasn't working as well on the preview. That might be happening because of the button styles bug @TheSisb was looking into. We're getting a lot of `undefined` styles on the docs site for some reason. Once that is fixed, the `margin-right` might fix as well.

I'll go ahead and change the `text-decoration` at the button level.

Reply:
yeah the undefined's are all autoprefixer based, everything that's not being auto prefixed seems to work just fine. Due to `Button` not exposing `classname` I full expect this particular code to not work though

Reply:
Right. That makes sense. Adding a nested style to the outer container to get the margin I want.

Reply:
It'll potentially break in the future, I was thinking more like:

```
<StyledButtonRow marginTop="space80" marginBottom="space80">
  <Box mr="space40">
    <Button as="a" href="/" size="default" variant="primary">
      Install Paste 0.1
    </Button>
  </Box>
  <Button as="a" href="/components" size="default" variant="secondary">
    View Components
  </Button>
</StyledButtonRow>```

Reply:
Ah I see. Worth changing now, or should I just run with what I have?

Reply:
Entirely up to you, but we should change it eventually

Reply:
Easy fix, so got it in. Had to add a `display` prop as well to make it all inline.